brave/browser-laptop
# Resolve package.json engine.node version conflicts

Fix #5616

No functional changes.

Auditors: @bbondy @mrose17

Test Plan:
1. `rm -rf node_modules`
2. `rm -rf ~/.electron/`
3. `npm install`
4. Confirm browser works.

Test Plan (yarn):
1. `rm -rf node_modules`
2. `rm -rf ~/.electron/`
3. `yarn cache clean`
4. `yarn install`
5. Confirm browser works.
